erts/etc/common/erlexec.c

```diff
@@ -2121,14 +2139,15 @@ initial_argv_massage(int *argc, char ***argv)
     if (vix == (*argc > 1 ? 2 : 0)) {
 	/* Only command line argv; check if we can use argv as it is... */
 	ac = *argc;
-	av = *argv;
+	av = argv;
 	for (ix = 1; ix < ac; ix++) {
 	    if (strcmp(av[ix], "-args_file") == 0) {
```

Comment:
I was wondering why Github CI got a segfault but I didn't locally, turns out it is because I have ERL_FLAGS set in my own environment... thanks for digging into this!